.gitignore

```diff
@@ -37,3 +37,5 @@
 /phpunit.xml
 /phpspec.yml
 /.web-server-pid
+
+/.idea/*
```

Comment:
It should not be a part of this PR. Please, add this folder to your global gitignore